ci/build_and_publish_docker_images.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 set -e
 
-# If not staging and master branch are existing
 export TAG="$CIRCLE_BRANCH"
 
+if [ "$CIRCLE_BRANCH" == "master" ]; then
+    export TAG="latest";
```

Comment:
@yann300 always master branch builds are tagged as latest (as that is default tag with docker) and we want users to always aim for latest stable build. 

Specific build releases can also be added and that is not covered here.